src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -176,6 +176,11 @@ class GlobalSelectionHeader extends React.Component {
       return true;
     }
 
+    // Update if `forceProject` changes
```

Comment:
how did this ever update in the first place?

Reply:
It didn't need to because forceProject was always the same